docs/src/inputs/dropdownMenu/dropdownMenu.jsx

```diff
@@ -10,6 +13,39 @@ import Main from '../../global/main';
 import { default as dropdownMenuDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/inputs/dropdownMenu/dropdownMenu';
 /* eslint-enable import/no-named-default, import/extensions */
 
+const children = [1, 2, 3, 4, 5, 6];
+const isOpen = true;
+const id = 'myId';
+const theme = 'dark';
+
+const dropDownMenuSample = `import React from 'react';
+
+import Loader from '../app/Elements/Loader.react';
+
+export default class TypeSample extends React.Component {    
+
+    constructor(props) {
+        super(props);        
+    }
+    
+    const children = [1, 2, 3, 4, 5, 6];
+    const isOpen = true;
+    const id = 'myId';
+    const theme = 'dark';
+
+    render() {
+        return (
+            <DropdownMenu
+                id={id}
+                isOpen={isOpen}
+                children={children}
+                onToggleOpen={() => console.log('Open!')}
```

Comment:
Should probably be like this (both in example source code string and live code):
```JavaScript
onToggleOpen={() => { console.log('Open!'); }}
```

Reply:
Fixed.

Reply:
I no longer see this file diffed at all.  Did we revert this change?
In any event, I suppose the matter is resolved.

Reply:
Yes, I reverted it. Many of these changes should have never happened.